datasets/shield.md

```diff
@@ -1,5 +1,121 @@
-# Work in Progress
+# Analyzing data from SHIELD studies
 
-This document is a work in progress.
-The work is being tracked in
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1358286)
+This article introduces the datasets that are useful for analyzing SHIELD studies.
+After reading this article,
+you should understand how to answer questions about
+study enrollment,
+identify telemetry from clients enrolled in an experiment,
+and locate telemetry from add-on studies.
+
+## Table of contents
+
+<!-- toc -->
+
+## Dashboards
+
+[Firefox Test Tube](https://firefox-test-tube.herokuapp.com/)
+shows live enrollment activity and a selection of browser activity metrics
+for active pref-flip (but not add-on) experiments.
+
+## Experiment slugs
+
+Each experiment is associated with a slug,
+which is the label used to identify the experiment to Normandy clients.
+The slug is also used to identify the experiment in telemetry.
+
+You can determine the slug for a particular experiment by consulting
+[Test Tube](https://firefox-test-tube.herokuapp.com/) (only for pref-flip experiments),
+[this summary table](https://metrics.mozilla.com/~sguha/report/normandy_recipes.html),
```

Comment:
There's also https://strategy-and-insights.mozilla.com/shield-studies/index.html

Reply:
Added this and Experimenter to the dashboards section -- neither of these actually displays slugs!